vortex-ffi/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "vortex-ffi"
+description = "Native C FFI bindings for Vortex"
+readme = "README.md"
+# This crate is not meant to be consumed by other Rust code but rather produces a static binary
+# that can be linked to by other languages.
+publish = false
+version.workspace = true
+homepage.workspace = true
+repository.workspace = true
+authors.workspace = true
+license.workspace = true
+keywords.workspace = true
+include.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+categories.workspace = true
+
```

Comment:
We should be consistent with how we do in other crates `version = { workspace = true }`. there's a lint that we need to apply to avoid it in the future